README.md

```diff
@@ -65,6 +65,39 @@ customElements.define('new-element', NewElement);
 
 See examples in the demo or try this live [JS bin](https://jsbin.com/zitotejimi/edit?html,output).
 
+## Uprading from 3.x
+
+The component has been migrated to LitElement and uses TypeScript now. This migration introduced two breaking changes.
+
+### Removed Polymer-specific `selection-changed` event used for 2-way bindings
+
+There were two events that notify about chart selection changes: `google-chart-select` and the Polymer-generated `selection-changed`.
+For consistency with other events (e.g. `google-chart-ready`), we keep only `google-chart-select`.
+
+Polymer components using 2-way binding for the selection should be updated to explicitly name the event - https://polymer-library.polymer-project.org/3.0/docs/devguide/data-binding#two-way-native. Note the `::google-chart-select` part.
+
+```diff
+- <google-chart selection="{{chartSelection}}"></google-chart>
++ <google-chart selection="{{chartSelection::google-chart-select}}"></google-chart>
+```
+
+LitElement components should use the `google-chart-select` event:
+
+```diff
+- <google-chart .selection="${chartSelection}" @selection-changed="${reactToChartSelection}"></google-chart>
++ <google-chart .selection="${chartSelection}" @google-chart-select="${reactToChartSelection}"></google-chart>
+```
+
+### Removed deprecated `google-chart-loader` component
+
+Its functionality is exported from the `loader.js` module:
```

Comment:
"Its functionality can be imported from the `loader.js` module or you may instead choose to use a `google.visualization.ChartWrapper` directly."

Reply:
Do we need an example of using a ChartWrapper?

Reply:
Done. Added a link to Google Charts docs about `ChartWrapper`.

The only part that is not available in `loader.js` is getting the constructor for the given chart type.